libs/infra-nest-server/src/lib/infra/health/health.controller.ts

```diff
@@ -0,0 +1,42 @@
+import { Controller, Get, Inject, Res } from '@nestjs/common'
+import {
+  HealthCheck,
+  HealthCheckService,
+  SequelizeHealthIndicator,
+} from '@nestjs/terminus'
+import { Response } from 'express'
+
+import { HealthCheckOptions, HealthCheckOptionsProviderKey } from './types'
+
+@Controller('health')
+export class HealthController {
+  constructor(
+    private readonly health: HealthCheckService,
+    private readonly db: SequelizeHealthIndicator,
+
+    @Inject(HealthCheckOptionsProviderKey)
+    private readonly healthCheckOptions: HealthCheckOptions,
+  ) {}
+
+  // route: /health/check
```

Comment:
Most of our nest servers reuse the `/liveness` path as the health check for kubernetes `readiness` probe.
We need to configure this in the service infra where we want to use this new health check route instead of the `/liveness` route.

But we need to update readiness probe script in kubernetes to check for the status property of the json response, today it is only checking for 200 ok response.